tokio-stream/src/timeout.rs

```diff
@@ -20,10 +21,14 @@ pin_project! {
     }
 }
 
+/// Error returned by `Timeout`.
+#[derive(Debug, PartialEq)]
+pub struct Elapsed(());
```

Comment:
This change moves the `Elapsed` error into this file, but I can't find the part of the diff that removes it from the `tokio::time::error` module.

Reply:
Its still used here, right? https://docs.rs/tokio/0.3.6/tokio/time/struct.Timeout.html#impl-Future

Reply:
So we now have two copies of it, but I don't think its that big of a deal in the end

Reply:
Ahh.. Right.. It's duplicated because it is now in two different crates.